contrib/bentoml/README.md

```diff
@@ -1,193 +1,209 @@
-# BentoML Yatai Stack
+# BentoML on Kubeflow
 
-[BentoML Yatai Stack](https://github.com/bentoml/yatai-deployment) is a series of components for deploying models/bentos to Kubernetes at scale
+Starting with the release of Kubeflow 1.7, BentoML provides a native integration with Kubeflow through [Yatai](https://github.com/bentoml/yatai-deployment). This integration allows you to package models trained in Kubeflow notebooks or pipelines as [Bentos](https://docs.bentoml.org/en/latest/concepts/bento.html), and deploy them as microservices in a Kubernetes cluster through BentoML's cloud native components and custom resource definitions (CRDs). This documentation provides a comprehensive guide on how to use BentoML and Kubeflow together to streamline the process of deploying models at scale.
 
 ## Requirements
 
 * Kubernetes 1.20 - 1.24
 
 ## Installation
 
-    * The yaml assumes you will install in kubeflow namespace
+Run the following command to install BentoML Yatai. Note that the YAML assumes you will install in kubeflow namespace.
 
 ```bash
 kustomize build bentoml-yatai-stack/default | kubectl apply -n kubeflow --server-side -f -
 ```
 
-## Upgrating
+## Upgrading
 
 See [UPGRADE.md](UPGRADE.md)
 
-# Design Proposal
-
 ## Why BentoML
 
 ![image](https://user-images.githubusercontent.com/861225/212856116-bf873dc8-7da3-4484-9f33-e401e34a82dc.png)
```

Comment:
Please include the images in the repository itself.

Reply:
Removed this image and moved the detailed example to the BentoML repo. The documentation here will link the full example in BentoML.